community/planning/cylinder_jwt.md

```diff
@@ -0,0 +1,227 @@
+# Cylinder JWT
+<!--
+  Copyright 2018-2020 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+## Summary
+[summary]: #summary
+
+This proposal adds a JSON Web Token (JWT) module to the Cylinder library.
+
+## Motivation
+[motivation]: #motivation
+
+Implementing a JWT feature in Cylinder allows for the use of its own signer
+implementations to provide signed tokens. This provides Cylinder users to make
+use of the same keys as an identity for use in both HTTP authentication tokens
+(via JWT) as well as the current comment use case of transaction signers.
+
+Existing JWT libraries do not allow for the flexibility to use custom signers.
+
+##Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+Cylinder JWT provides a set of API's to both generate a JWT-like token and
+cryptographically validate its contents.
+
+The header will follow the standard minimally consist of a type (`"typ"`) of
+`"cylinder+jwt"`, and an algorithm (`"alg"`) value that depends on the Signer
+implementation.
+
+The use of the term JWT-like is necessary as the algorithms currently provided
+by Cylinder are not part of the standard set.  The resulting token will still
+follow the format of
+
+```
+<base64 encoded bytes of the utf8 string of the header json>.\
```

Comment:
This is displayed as-is with the backslashes - is that intentional?

Reply:
Yes, otherwise, it scrolls of the screen.  This makes it a bit more readable to the user.  I could put that in quotes, so that it is also clear that it is an string value.